pkg/engine/task/task.go

```diff
@@ -151,9 +151,11 @@ var (
 )
 
 func validPipeFile(pf PipeFile) error {
-	if pf.File == "" {
-		return fmt.Errorf("task validation error: pipe file is empty: %v", pf)
+	fset, efset := pf.File != "", pf.EnvFile != ""
```

Comment:
```suggestion
	fset := pf.File != ""
	efset := pf.EnvFile != ""
```
I don't know why the syntax with `,` is allowed, but I find it hard to read and not worth saving a LOC...